apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -1,5 +1,6 @@
 const NotificationMessage = {
 	CONTRIBUTOR_MERGE_SUCCESS: "Contributor was successfully merged.",
```

Comment:
Oh, and let's then rename this 
```suggestion
	CONTRIBUTOR_MERGE_SUCCESS: "Contributors were successfully merged.",
```
since we are merging two contributors together